p2p/switch.go

```diff
@@ -486,7 +483,10 @@ func (sw *Switch) listenerRoutine(l Listener) {
 	// cleanup
 }
 
-func (sw *Switch) addInboundPeerWithConfig(conn net.Conn, config *PeerConfig) error {
+func (sw *Switch) addInboundPeerWithConfig(
+	conn net.Conn,
```

Comment:
Is this the result of default formatter? or you manually splitting the line

Reply:
This is how arguments are split when ranged over multiple lines.